README.md

```diff
@@ -136,7 +136,7 @@ module.exports = {
         test: /\.less$/,
         use: ExtractTextPlugin.extract({
           fallbackLoader: 'style-loader',
-          loaders: [
```

Comment:
Man, this is acceptable!!! This example is loading the webpack loader with 'rules` of webpack 2. All this examples on README are about webpack 2. Take a time for read little more about webpack 2 features.